contributors/devel/pull-requests.md

```diff
@@ -81,21 +79,24 @@ For PRs with a release note:
     Your release note here
     ```
 
-For PRs without a release note:
+For PRs that require additional action from users switching to the new release, include the string "action required" (case insensitive) in the release note:
+
+    ```release-note
+    action required: your release note here
+    ```
+
+For PRs that don't need to be mentioned at release time, just write "NONE" (case insensitive):
 
     ```release-note
     NONE
     ```
 
+The `/release-note-none` comment command can still be used as an alternative to writing "NONE" in the release-note block if it is left empty.
+
 To see how to format your release notes, view the [PR template](https://github.com/kubernetes/kubernetes/blob/master/.github/PULL_REQUEST_TEMPLATE.md) for a brief example. PR titles and body comments can be modified at any time prior to the release to make them friendly for release notes.
```

Comment:
>To see how to format your release notes, view the [PR template](https://github.com/kubernetes/kubernetes/blob/master/.github/PULL_REQUEST_TEMPLATE.md) for a brief example.

If we provide release note template in both places, then we don't need this xref.